hazelcast/src/main/java/com/hazelcast/config/MerkleTreeConfig.java

```diff
@@ -214,17 +182,14 @@ public boolean equals(Object o) {
         if (enabled != that.enabled) {
             return false;
         }
-        if (depth != that.depth) {
-            return false;
-        }
-        return mapName != null ? mapName.equals(that.mapName) : that.mapName == null;
+        return depth == that.depth;
```

Comment:
`MerkleTreeConfig` lacks a test for `equals`/`hashCode`, can you add one?

Reply:
Will do!